joomla/joomla-cms
# [4.0] [DRAFT] Mod_quickicon normalization

Pull Request for Issue # .

### Summary of Changes
Icon array is titled "image".  This changes it to icon to be more consistent with its function.


### Testing Instructions
inspect 4.0-dev dashboard.  Verify quickicons are showing.
![image](https://user-images.githubusercontent.com/1850089/92444025-eeeaf800-f177-11ea-83e4-5c8ba3503180.png)
apply pr.
verify icons still display properly.

### Actual result BEFORE applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/92444063-fad6ba00-f177-11ea-9889-298ab1ade247.png)



### Expected result AFTER applying this Pull Request
![image](https://user-images.githubusercontent.com/1850089/92444066-fc07e700-f177-11ea-8e7e-2ba872276452.png)



### Documentation Changes Required
b/c as it may require third party devs to change the $displayData['image'] to $displayData['icon']
